paths/workflow_runs.yaml

```diff
@@ -162,4 +162,35 @@
               schema:
                 $ref: ../schemas/tasks/task.yaml
         default:
-          $ref: ../responses/default_error.yaml
\ No newline at end of file
+          $ref: ../responses/default_error.yaml
+    post:
+      summary: Completes a Send / Receive Data task.
+      operationId: complete_task
+      parameters:
+        - name: workflow_run_id
+          in: path
+          required: true
+          description: The unique identifier of the Workflow Run to which the Task belongs.
+          schema:
+            type: string
+        - name: task_id
+          in: path
+          required: true
+          description: The identifier of the Task you want to complete.
+          schema:
+            type: string
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                data:
+                  type: object
+                  description: The Task completion payload.
+      responses:
+        "204":
```

Comment:
Small detail sorry, forgot to mention but the return code for this was update to be a 200 with an empty body {}. This was to account for a limitation in the sdk and also to leave it ready for a future where we may want to return something as output of this call.